src/panel/poi/PoiPanel.jsx

```diff
@@ -224,20 +224,22 @@ export default class PoiPanel extends React.Component {
   }
 
   renderContent = (poi, { size: panelSize, isMobile }) => {
-    if (isMobile && panelSize === 'minimized') {
-      return <div className="poi_panel__content">
-        <PoiItem poi={poi} withAlternativeName />
-      </div>;
-    }
-
     return <div className="poi_panel__content">
-      <PoiItem
-        poi={poi}
-        withAlternativeName
-        withOpeningHours
-        className="u-mb-20"
-        onClick={this.center}
-      />
+      {isMobile && panelSize === 'minimized' ?
+        <PoiItem
+          poi={poi}
+          withAlternativeName
+          className="u-mb-20"
+        />
+        :
+        <PoiItem
+          poi={poi}
+          withAlternativeName
+          withOpeningHours
+          className="u-mb-20"
+          onClick={this.center}
+        />
+      }
```

Comment:
the only difference between the two components instances are the props `withOpeningHours` and `onClick`.
You can just add conditions to these two props to check if it's `isMobile && panelSize === 'minimized'`

Reply:
You can even spare the `isMobile` test, as `panelSize` will never equal `minimized` if we aren't on mobile.